### ISSUE TITLE
Row Selection Does Not Support Enter Key Event

### ISSUE THREAD

**Issue by jmmadsen** [react-bootstrap-table/react-bootstrap-table2#1256]
When a table has the row selection column, you are able to navigate using the `tab` key, which is great for keyboard accessibility. However, you are unable to trigger any selection events using the `enter` key, which means keyboard accessibility is only halfway working.

Essentially, `selectRow.onSelect` and `selectRow.onSelectAll` do **NOT** listen for the `enter` key, which breaks keyboard accessibility.

**To Reproduce**
I tried all the story examples, but none worked with the `enter` key. -> https://react-bootstrap-table.github.io/react-bootstrap-table2/storybook/index.html?selectedKind=Row%20Selection&selectedStory=Single%20Selection&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

**codesandbox**
I can give a code sandbox if necessary, but I am really not deviating far from the story examples. I am simply running through and trying to make my app 508 compliant.

**Comment by jmmadsen** [comment]
Just figured out you can use the space bar to select. Might want to add the feature in a future release to also use the enter key, but I will close this for now.
